apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -143,6 +149,11 @@ defmodule Aecore.Chain.Worker do
 
   ## Server side
 
+  def handle_call(:clear_state, _from, state) do
+    {:ok, new_state, _} = init("")
```

Comment:
looks weird with `""` I know it doesn't matter but maybe choose something else

Reply:
I'll look into this